hypervisor/src/kvm/mod.rs

```diff
@@ -1030,6 +1033,17 @@ impl cpu::Vcpu for KvmVcpu {
                     Ok(cpu::VmExit::MmioWrite(addr, data))
                 }
                 VcpuExit::Hyperv => Ok(cpu::VmExit::Hyperv),
+                VcpuExit::Unsupported(r) => {
```

Comment:
You could write this as `VcpuExit::Unsupported(KVM_EXIT_TDX)` i.e. a separate match line.

Reply:
Ah yes this will be a bit more elegant.